src/strings/en-us.json

```diff
@@ -1467,7 +1473,6 @@
     "TabDashboard": "Dashboard",
     "TabLatest": "Recently Added",
     "TabLogs": "Logs",
-    "TabMusic": "Music",
```

Comment:
Is this key change necessary?

Reply:
Yes, because not all library keys have the tab naming convention.